### PR TITLE
xschem: fix paths

### PR DESCRIPTION
**Pull request by proppy**
- fix symbols path
- fix models extension

/cc @atorkmabrains @StefanSchippers

### CODE REVIEW

File: cells/xschem/tests/test_vnpn_10x10.sch [hunk 1]
```diff
@@ -69,12 +69,12 @@ lab=C}
 C {devices/code_shown.sym} 30 -200 0 0 {name=MODELS only_toplevel=true
 format="tcleval( @value )"
 value="
-.include $::180MCU_MODELS/design.spice
-.lib $::180MCU_MODELS/sm141064.spice typical
-.lib $::180MCU_MODELS/sm141064.spice res_typical
-.lib $::180MCU_MODELS/sm141064.spice moscap_typical
-.lib $::180MCU_MODELS/sm141064.spice bjt_typical
-* .lib $::180MCU_MODELS/sm141064.spice res_statistical
+.include $::180MCU_MODELS/design.n.ngspice
```
Comment (proppy) [inline cells/xschem/tests/test_vnpn_10x10.sch]: this should be `.ngspice`

**Comment by QuantamHD** [comment]
This ready to go @proppy

**Review by atorkmabrains — APPROVED** [review]
LGTM

**Comment by atorkmabrains** [comment]
@proppy @mithro Are you going to merge?

**Pull request merged**